ai-platform-unified/notebooks/unofficial/migration/UJ10 unified Custom Training Prebuilt Container SKLearn.ipynb

```diff
@@ -187,13 +187,13 @@
     "#### Region\n",
     "\n",
     "You can also change the `REGION` variable, which is used for operations\n",
-    "throughout the rest of this notebook.  Below are regions supported for AI Platform (Unified). We recommend when possible, to choose the region closest to you.\n",
+    "throughout the rest of this notebook.  Below are regions supported for Vertex AI. We recommend when possible, to choose the region closest to you.\n",
     "\n",
     "- Americas: `us-central1`\n",
     "- Europe: `europe-west4`\n",
     "- Asia Pacific: `asia-east1`\n",
     "\n",
-    "You cannot use a Multi-Regional Storage bucket for training with AI Platform. Not all regions provide support for all AI Platform services. For the latest support per region, see [Region support for AI Platform (Unified) services](https://cloud.google.com/ai-platform-unified/docs/general/locations)\n",
+    "You cannot use a Multi-Regional Storage bucket for training with Vertex AI. Not all regions provide support for all Vertex AI services. For the latest support per region, see [Region support for Vertex AI services](https://cloud.google.com/ai-platform-unified/docs/general/locations)\n",
```

Comment:
Change `https://cloud.google.com/ai-platform-unified/docs/general/locations` to `https://cloud.google.com/vertex-ai/docs/general/locations`

Also do for other affected links.